src/EnergyPlus/CrossVentMgr.cc

```diff
@@ -98,7 +96,7 @@ namespace CrossVentMgr {
     Real64 constexpr CrecFlow1(0.415); // First correlation constant for the recirculation flow rate
     Real64 constexpr CrecFlow2(0.466); // Second correlation constant for the recirculation flow rate
 
-    void ManageUCSDCVModel(EnergyPlusData &state,
+    void ManageCrossVent(EnergyPlusData &state,
```

Comment:
With all due respect to UCSD, there is no need to embed `UCSD` into variable and function names. I don't add `Roth` to names.